k8s/namespaces/bsp/bulk-scan-orchestrator/bulk-scan-orchestrator.yaml

```diff
@@ -13,8 +13,8 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/bulk-scan/orchestrator:prod-9e725ad-20211004092631
+      image: hmctspublic.azurecr.io/bulk-scan/orchestrator:prod-9e725ad-20211004092631   #{"$imagepolicy": "flux-system:bulk-scan-orchestrator"}
       smoketests:
-        image: hmctspublic.azurecr.io/bulk-scan/orchestrator-test:prod-9e725ad-20211004092631
+        image: hmctspublic.azurecr.io/bulk-scan/orchestrator-test:prod-9e725ad-20211004092631   #{"$imagepolicy": "flux-system:orchestrator-test"}
```

Comment:
Seems to be bulk-scan-orchestrator-orchestrator-test currently

Reply:
Policy name label updated.